Orm/Xtensive.Orm.PostgreSql/Orm.Providers.PostgreSql/SqlCompiler.cs

```diff
@@ -24,22 +24,31 @@ protected override SqlProvider VisitFreeText(FreeTextProvider provider)
       var binding = new QueryParameterBinding(stringTypeMapping,
         provider.SearchCriteria.Invoke, QueryParameterBindingType.Regular);
 
-      SqlSelect select = SqlDml.Select();
+      var select = SqlDml.Select();
       var realPrimaryIndex = provider.PrimaryIndex.Resolve(Handlers.Domain.Model);
       var index = realPrimaryIndex.ReflectedType.Indexes.PrimaryIndex;
       var query = BuildProviderQuery(index);
       var table = Mapping[realPrimaryIndex.ReflectedType];
-      var fromTable = SqlDml.FreeTextTable(table, binding.ParameterReference, table.Columns.Select(column => column.Name).AddOne(rankColumnName).ToList());
+      var fromTable = SqlDml.FreeTextTable(table, binding.ParameterReference,
+        table.Columns.Select(column => column.Name).AddOne(rankColumnName).ToList());

```

Comment:
Append can be used instead of AddOne from netStandard 2.0

Reply:
Fixed.